R/mod_review_form_tbl_fct_helpers.R

```diff
@@ -1,3 +1,108 @@
+
+#' Get form table
+#'
+#' @param form_data A data frame with the study data of the respective form.
+#' @param form_review_data A data frame with he review data of the form.
+#' @param form A character string with the name of the form.
+#' @param form_items Named character vector with all form_items to display.
+#' @param active_subject A character string with the active subject id.
+#' @param is_reviewed A logical, indicating whether all items of the entire form
+#'   for the active subject id are reviewed.
+#' @param is_SAE A logical, indicating whether the form is a SAE form. If TRUE,
+#'   will make some adjustments to the columns to display.
+#' @param id_cols Columns that identify a unique row in the data.
+#'
+#' @keywords internal
+#'
+get_form_table <- function(
+    form_data,
+    form_review_data,
+    form,
+    form_items,
+    active_subject,
+    is_reviewed = NULL,
+    is_SAE = NULL,
```

Comment:
```suggestion
    is_reviewed = FALSE,
    is_SAE = FALSE,
```
Not sure why we would set the defaults to NULL when the value is supposed to be a logical.

Reply:
That is probably better indeed. Note that `is_reviewed` can be `NULL`/indetermined, but that is indeed not a reason to set the default at `NULL`, and not a reason for the default of `is_SAE` to be `NULL`.

Reply:
Maybe it makes more sense for `is_reviewed = NULL` since that's tracking the checkbox update item.